lib/attribute_handlers/discriminator.rb

```diff
@@ -0,0 +1,19 @@
+require_relative "../ref_schema_parser"
+
+module OpenApi2JsonSchema
+  module AttributeHandlers
+    class Discriminator
+      def call(discriminator_schema)
+        raise "discriminator schema must be a Hash" unless discriminator_schema.is_a?(Hash)
+        raise "discriminator schema must contain mapping" unless discriminator_schema['mapping']
+
+        mapping = discriminator_schema["mapping"]
```

Comment:
You can use `fetch` instead:
```suggestion
        mapping = discriminator_schema.fetch("mapping")
```